src/main/kotlin/uk/gov/justice/digital/hmpps/incidentreporting/dto/nomis/Mapping.kt

```diff
@@ -85,17 +85,17 @@ fun Report.addNomisCorrectionRequests(correctionRequests: Collection<NomisRequir
 }
 
 fun Report.addNomisQuestions(questions: Collection<NomisQuestion>) {
-  questions.sortedBy { it.sequence }.forEach { question ->
+  questions.forEach { question ->
     val dataItem = this.addQuestion(
       code = question.questionId.toString(),
+      sequence = question.sequence - 1,
```

Comment:
Why is the sequence reduced by one?

Reply:
NOMIS starts from 1

Reply:
so this is simply to try and have sequences starting from 0 in our DB 🤔 - but nothing to do with the data problems 👍

Reply:
well we gotta put _something_ there now that ordering must be manually managed